src/main/resources/com/xceptance/xlt/report/evaluation/evaluation-schema.json

```diff
@@ -0,0 +1,259 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/Xceptance/XLT/someschema.json",
```

Comment:
Do we want to replace `someschema` with something more specific?

Reply:
Done.